src/main/java/com/stripe/net/FormEncoder.java

```diff
@@ -78,6 +78,16 @@ public static String createQueryString(
             .collect(Collectors.toList()));
   }
 
+  /**
+   * Returns a list of flattened parameters for the given map of parameters.
```

Comment:
Would it be possible to explain what "flatten" in this context means?

Reply:
👍 I added a longer comment.